### DISCUSSION TITLE
Possibility to request an image from a recording at a specific time

### DISCUSSION THREAD

**Discussion by flispy** [blakeblackshear/frigate#14536] [Ask A Question] [answered]
Hi,

It would be great if there is a possibility to get an image over the API for a specific time. Why? Not every object is detected via frigate and I want to get the current image from the recording at a specific time - eg. the door has been opened and I want to get the image once it has happened.
Is there any possibility to get it via an URL like [[?](http://192.168.xxx.xx:5000//api/cam/recordings/1729584000/snapshot.jpg)](http://192.168.xxx.xx:5000//api/cam/recordings/1729584000/snapshot.jpg)

**Comment by hawkeye217** [comment]
See the docs: https://docs.frigate.video/integrations/api#get-apicamera_namerecordingsframe_timesnapshotpng

**Reply by flispy** [reply]
> See the docs: https://docs.frigate.video/integrations/api#get-apicamera_namerecordingsframe_timesnapshotpng

I already tried it without success ☹️ do I use a wrong timestamp in my url?

**Reply by flispy** [reply]
error message is: "{"message":"Recording not found at 1729584000.0","success":false}"

**Reply by hawkeye217** [reply]
The API will return an error message if your request is invalid.

**Reply by hawkeye217** [reply]
What is your Frigate recording config? You are just requesting a timestamp where Frigate does not have a recording available.

**Reply by flispy** [reply]
Recording is definitely enabled and can be used on frontend to go back to other timestamps

**Reply by hawkeye217** [reply]
What recording mode are you using in your Frigate config?

**Reply by flispy** [reply]
record:
  enabled: true
  expire_interval: 180

**Reply by hawkeye217** [reply, marked as answer]
You should read the [recording docs](https://docs.frigate.video/configuration/record/) to ensure you save footage for the times you're wanting to view.

**Reply by flispy** [reply]
> You should read the [recording docs](https://docs.frigate.video/configuration/record/) to ensure you save footage for the times you're wanting to view.

thanks a lot. I only saved events to disk instead of doing it for all. I missed:
retain:
        days: 1
        mode: all

**Reply by hawkeye217** [reply]
Great. Happy to help.
